docs/accessing-the-cluster.md

```diff
@@ -203,5 +301,40 @@ monitoring-influx-grafana-controller-0133o   10.244.3.4          influxdb
                                                                  grafana                 kubernetes/heapster_grafana:v0.4
 ```
 
+##<a name="somanyproxies"></a>So Many Proxies
+There are several different proxies you may encounter when using kubernetes:
+  1. The [kubectl proxy](#kubectlproxy)
+    - runs on a users desktop or in a pod
+    - proxies from a localhost address to the kubernetes apiserver.
```

Comment:
Suggest either consistent full sentences (Capital and trailing '.') or consistent fragments (no capital, no '.').